nimble/host/src/ble_l2cap_priv.h

```diff
@@ -62,6 +62,7 @@ typedef int ble_l2cap_rx_fn(struct ble_l2cap_chan *chan);
 
 struct ble_l2cap_chan {
     SLIST_ENTRY(ble_l2cap_chan) next;
+    bool disconnecting;
```

Comment:
what about using `flags` instead?

Reply:
yeah, it was too quick fix and I forgot about our flags. Changing it. Thanks